### PR TITLE
[Fix](workloadgroup)reset remote scan thread num

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

File: be/src/vec/exec/scan/scanner_scheduler.h [hunk 1]
```diff
@@ -127,6 +127,15 @@ class SimplifiedScanScheduler {
         return Status::OK();
```
Comment (yiguolei) [inline be/src/vec/exec/scan/scanner_scheduler.h:127]: This start method is useless now. Could use the new start method.

File: be/src/runtime/workload_group/workload_group.cpp [hunk 2]
```diff
@@ -386,10 +386,19 @@ void WorkloadGroup::upsert_task_scheduler(WorkloadGroupInfo* tg_info, ExecEnv* e
...
+                        ? config::doris_max_remote_scanner_thread_pool_thread_num
+                        : std::max(512, CpuInfo::num_cores() * 10);
+        remote_max_thread_num = std::max(_remote_thread_pool_max_size,
+                                         config::doris_scanner_thread_pool_thread_num);
+
         std::unique_ptr<vectorized::SimplifiedScanScheduler> remote_scan_scheduler =
                 std::make_unique<vectorized::SimplifiedScanScheduler>("RScan_" + tg_name,
                                                                       cg_cpu_ctl_ptr);
-        Status ret = remote_scan_scheduler->start();
+        Status ret =
```
Comment (yiguolei) [inline be/src/runtime/workload_group/workload_group.cpp:399]: why  min == max??

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Pull request merged**